app/Http/Controllers/Api/ApiLinksController.php

```diff
@@ -0,0 +1,108 @@
+<?php
+
+namespace App\Http\Controllers\Api;
+
+use Illuminate\Http\Request;
+
+use App\Models\Link;
+
+class ApiLinksController extends ApiController
```

Comment:
I think we should better rename it, and host in this class any other bulk links management operations. So eg. ApiLinksEditController would host all operations onto multiple Links at once, whereas ApiListController could host operations on one Link at a time.